### PR TITLE
Added snippet to play video using javaFX

### LINKED ISSUE #109: Play video file
The method can play a video file

### PR DESCRIPTION
**Pull request by RohitPatel1122**
- [x] Added code to play video via 3rd Party library JavaFX
- [x] Added required test case to validate valid and invalid file format
- [x] tested that build is passing locally.

Fix #109

### CODE REVIEW

**Comment by RohitPatel1122** [comment]
@iluwatar Build for some reason is taking long time to finish. Kindly review

**Comment by iluwatar** [comment]
Please resolve the conflicts @RohitPatel1122

File: src/main/java/media/PlayVideoSnippet.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+public class PlayVideoSnippet extends Application {
+  private Media media = null;
+  private MediaPlayer mediaPlayer = null;
+  private MediaView mediaView = null;
+  private Scene scene = null;
+
+  public static void main(String[] args) {
+    Application.launch(PlayVideoSnippet.class, args);
+  }
```
Comment (iluwatar) [inline src/main/java/media/PlayVideoSnippet.java:50]: We need to document the expected parameters

File: src/main/java/media/PlayVideoSnippet.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    mediaPlayer = new MediaPlayer(media);
+    mediaPlayer.setAutoPlay(true);
+    mediaPlayer.setOnEndOfMedia(() -> Platform.exit());
+    mediaView = new MediaView(mediaPlayer);
+    scene = new Scene(new Group(), media.getWidth(), media.getHeight());
+    stage.setScene(scene);
+    stage.setTitle("Video Player");
+    ((Group) scene.getRoot()).getChildren().add(mediaView);
+    stage.show();
+  }
```
Comment (iluwatar) [inline src/main/java/media/PlayVideoSnippet.java:75]: - Use `var` for local variables where possible
- Strive to show the minimal example with as few lines as possible (while keeping the readability)

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Comment by iluwatar** [comment]
With the new code, our CI seems to hang. Need to figure out what to do with that...

File: src/main/java/media/PlayVideoSnippet.java (hunk 1 above)
Comment (RohitPatel1122) [reply src/main/java/media/PlayVideoSnippet.java:51]: have added what is expected as args

**Comment by iluwatar** [comment]
I think the code looks good, but we need to make the CI build pass. I tried to add JavaFX support in https://github.com/iluwatar/30-seconds-of-java/pull/158. Could you rebase this branch on top of `master` to check if it works now @RohitPatel1122 ?

**Comment by RohitPatel1122** [comment]
I have resolved the merge conflicts. But not sure, for what reason the build is failing. Kindly look into it.

**Comment by iluwatar** [comment]
I needed to revert the CI change since it didn't work. Do you have any suggestions how to make Github Actions work with JavaFX?

**Comment by RohitPatel1122** [comment]
> I needed to revert the CI change since it didn't work. Do you have any suggestions how to make Github Actions work with JavaFX?

What I understand is, Eclipse distribution `temurin` does not provide jdk+fx package. Check [this](https://adoptium.net/en-GB/temurin/releases/).
While Zuul give it . Check [this](https://www.azul.com/downloads/?version=java-17-lts&package=jdk-fx)

**Pull request closed without merging**